backend/apps/chat/consumers.py

```
@@ -99,8 +147,13 @@ async def connect(self):
                 self.group_name,
                 {
                     "type": "presence_joined",
+
+                    "username": getattr(self.user, "username", ""),
+                    "user_id": user_id,
+
                     "username": self.user.username,
```

Ruff (0.15.21)

[error] Dictionary key literal `"username"` repeated

(F601)